### PR TITLE
Add ansible configuration for ci server

### PR DESCRIPTION
**Pull request by everestmz**
cc: @google/clusterfuzz 
Adds the initial ansible configuration files - allows for deploying and running the script successfully. Logging with `svlogd` still needs some tweaking to work properly.

### CODE REVIEW

File: ci/ansible/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - name: Add host to groupname
+      add_host: hostname={{ gce.instance_data[0].public_ip }}
+                groupname=continuous-integration-instances
+- name: Configure users and dependencies
+  hosts: continuous-integration-instances
+  connection: ssh
+  become: true
+  tasks:
+    - group:
+        name: clusterfuzz
```
Comment (tanin47) [inline ci/ansible/playbook.yml]: Add `state: present` even though it might be the default.

File: ci/ansible/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - file: path="{{item}}"
+            state=directory
+            owner=root
+            group=root
+            mode='u=rwx,g=rwx,o=rwx'
+      with_items:
+        - '/etc/sv/python-daemon'
+        - '/etc/sv/python-daemon/log'
+        - '/etc/runit/insanity/log'
+        - '/etc/monit'
```
Comment (tanin47) [inline ci/ansible/playbook.yml]: Remove monit.

File: ci/ansible/playbook.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        mode: 'u=rwx,g=rx,o=rx'
+    - file: path="{{item}}"
+            state=directory
+            owner=root
+            group=root
+            mode='u=rwx,g=rwx,o=rwx'
+      with_items:
+        - '/etc/sv/python-daemon'
+        - '/etc/sv/python-daemon/log'
+        - '/etc/runit/insanity/log'
```
Comment (tanin47) [inline ci/ansible/playbook.yml]: Remove this.

File: ci/ansible/playbook.yml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    - copy: src={{item.src}}
+            dest={{item.dest}}
+            force=true
+            owner=root
+            group=root
+            mode='u=rwx,g=rwx,o=rwx'
+      with_items:
+        - {'src': 'python_daemon_run', 'dest': '/etc/sv/python-daemon/run'}
+        - {'src': 'python_daemon_log_run', 'dest': '/etc/sv/python-daemon/log/run'}
+        - {'src': 'monitrc', 'dest': '/etc/monit/monitrc'}
```
Comment (tanin47) [inline ci/ansible/playbook.yml]: Remove monit

File: .gitignore [hunk 5]
```diff
@@ -6,3 +6,7 @@
 /build
 /dist
 /htmlcov
+/ci/ansible/*.json
```
Comment (tanin47) [inline .gitignore]: I can see now that running ansible from the root is not ideal, especially, when we have a local ENV.

Let's run from where ENV is.

Move this content to `.gitignore` in `/ci/ansible` as well.

Moving forward, we will have multiple ENVs in this repo. This is something we want to do in clusterfuzz as well. Python's import conflict is getting crazy.

**Comment by everestmz** [comment]
Thank you, updated!

File: ci/ansible/.gitignore [hunk 6]
```diff
@@ -0,0 +1,4 @@
+*.json
```
Comment (tanin47) [inline ci/ansible/.gitignore]: Need to add slash in front of all these lines.

Comment (tanin47) [inline ci/ansible/.gitignore]: Also, let's move the credential file to `/auto/FuzzInfrastructure/ClusterFuzz/credentials`.

**Pull request merged**